src/XamlStyler/Options/StylerOptions.cs

```diff
@@ -356,6 +356,29 @@ public string ConfigPath
         [Browsable(false)]
         public bool SuppressProcessing { get; set; }
 
+        private string endOfLine;
+
+        [Category("Misc")]
+        [RefreshProperties(RefreshProperties.All)]
+        [Description("Defines end of line character. Specify 'lf' or 'crlf'; otherwise, default character of the host will be used.")]
+        [JsonProperty("EndOfLine", DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        [DefaultValue("")]
+        public string EndOfLine
+        {
+            get { return endOfLine; }
+            set
+            {
+                endOfLine = value;
+                NewLine = string.Equals("crlf", value, StringComparison.InvariantCultureIgnoreCase) ? "\r\n"
+                  : string.Equals("lf", value, StringComparison.InvariantCultureIgnoreCase) ? "\n"
+                  : Environment.NewLine;
+            }
+        }
+
+        [JsonIgnore]
```

Comment:
Similar to ```UseVisualStudioIndentSize``` this also needs the following so that it does not appear in the Visual Studio options UI:

```
[DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
[Browsable(false)]
```